geomstats/learning/frechet_mean.py

```diff
@@ -305,12 +309,19 @@ def __init__(self, metric,
                  point_type='vector',
                  method='default',
                  verbose=False):
+        error.check_parameter_accepted_values(
```

Comment:
We/I should probably factorize this into a `check_point_type` method, or the standalone decorator that also checks if point_type is None.